pkg/stream/edgedexecconnection.go

```diff
@@ -93,16 +93,15 @@ func (e *EdgedExecConnection) Serve(tunnel SafeWriteTunneler) error {
 		}
 	}()
 
+	var data [256]byte
 	for {
 		select {
 		case <-stop:
 			klog.V(6).Infof("receive stop single, so stop exec scan ...")
 			return nil
 		default:
 		}
-		data := make([]byte, 256)
-
-		n, err := con.Read(data)
+		n, err := con.Read(data[:])
```

Comment:
?

Reply:
There is no need to alloc memory for Msg-data every time in the loop. always use `var data [256]byte`

Reply:
Why [:]?

Reply:
Reduce small memory allocation and reduce GC time

Reply:
data is an Array type, not a Slice, need  [:] as to slice.

Reply:
Thanks for you fixing! :) And I'm not sure whether `con.Read(data[:])`  will refresh the whole byte in `data` every time? Or there wil be dirty byte in `data`. @wzshiming

Reply:
This is reference passing, similar to pointer passing @fisherxu

Reply:
@fisherxu 
`n, err := con.Read(data[:])`, will write `n` length data in `data[:]`, below use `data[:n]`.

Reply:
Make sense.